amt/locale/nl_NL/LC_MESSAGES/messages.po

```diff
@@ -16,7 +16,7 @@ msgstr ""
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=utf-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"Generated-By: Babel 2.16.0\n"
```

Comment:
How come this version is going down?

Reply:
Hmmm that's peculiar indeed. I am not sure why.